crates/tx-sitter-client/src/data.rs

```diff
@@ -58,15 +58,36 @@ pub struct GetTxResponse {
     // Sent tx data
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub tx_hash: Option<H256>,
-    pub status:  TxStatus,
+    pub status:  Option<TxStatus>,
```

Comment:
Do we want to add `#[serde(default, skip_serializing_if = "Option::is_none")]` here?

Reply:
Good catch, thanks!